charts/gardener/controlplane/charts/runtime/templates/scheduler/vpa.yaml

```
@@ -11,4 +11,10 @@ spec:
     name: gardener-scheduler
   updatePolicy:
     updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+      - containerName: '*'
+        minAllowed:
+          cpu: 10m
+          memory: 20Mi
```

```suggestion
          memory: 25Mi
```

ref https://github.com/gardener/gardener/pull/2164

---

You mean to take into account that after a restart the cpu usage will go up for a short period?

---

Yes. If the VPA scales down the scheduler to `10m` CPU limits then the scheduler might not be able to start anymore (that's what we've seen for some extension controllers).

---

makes sense to me.

---

I checked grafana again and especially after checked after new instances got created. Seems like the currently configured CPU is enough for that  (and we do not need 50). I would suggest to start like this and adapt on the way if necessary. WDYT?